Overview.bs

```diff
@@ -609,9 +607,14 @@ the encoded bytes at the start of every iteration to pick up any changes made in
     <td>The largest entry index encoded in this table.</td>
   </tr>
   <tr>
-    <td>uint32</td>
+    <td>uint16</td>
+    <td><dfn for="Format 1 Patch Map">maxGlyphMapEntryIndex</dfn></td>
+    <td>The largest [=Glyph Map=] entry index encoded in this table. Must be less than or equal to maxEntryIndex.</td>
+  </tr>
+  <tr>
+    <td>uint24</td>
     <td><dfn for="Format 1 Patch Map">glyphCount</dfn></td>
-    <td>Number of glyphs that mappings are provided for. Must match the numGlyphs field from [[open-type/maxp|maxp]].</td>
+    <td>Number of glyphs that mappings are provided for. Must match the numGlyphs field from either [[open-type/maxp|maxp]] or MAXP (if present). If both maxp and MAXP are present then, it must match the numGlyphs field in MAXP.</td>
```

Comment:
Are we referencing an ISO Working or Committee Draft?

Reply:
This may be a tricky question. Do W3C standards often reference "working" standards from other organizations? 

In any case I think it would be strange to reference that draft. I believe https://www.mpeg.org/wp-content/uploads/mpeg_meetings/146_Rennes/w23797.zip is not password-protected, but I'm not sure how stable the link will be.

Reply:
This would be a good thing to discuss at our next call, I'm curious to get Vlad's thoughts.

I believe that we don't want to reference (in a normative way) anything that's a work in progress (eg. we have the same issue with the shared brotli spec). So maybe this is better put in as a note about possibly needing to integrate with beyond64k in the future (and we'll leave the field a u24 to support that).

Reply:
Yeah -- all of that stuff is (at least) months away from being an official part of the specification. Maybe years.

Reply:
(BTW if we just want a reference point for potentially having more than 64K glyphs, CFF2 already supports that. It's not very useful in practice because, e.g., hmtx doesn't, but it's a reference point.)

Reply:
Reworked the text, now it only depends on maxp but allows the glyph count value to be greater than 65535 if maxp.numGlyphs is 65535. This will allow >16 bit glyph count to work without adding a dependency on MAXP. Once the boring expansion stuff is more official we can come back and update the requirements to also mention MAXP here.

Reply:
Updated the wording here again based on the discussion from todays working group call. Please take another look.